src/midi/midi.ts

```diff
@@ -1,7 +1,7 @@
 import { WebMidi } from "webmidi";
-import { ParameterType, ParameterHash, Parameter } from "../ParameterHash";
+import { Parameter } from "../ParameterHash";
 
-const MESSAGE_NUMBER = 176;
+const PARAM_CHANGE_MESSAGE = 179;
```

Comment:
So this value doesn't actually replace MESSAGE_NUMBER like you might think at a first glance. It's actually a new magic number that is sent as the message ID on a midi change event. That ID is for changes to control params. We need to filter by this to avoid message from other parts of the synth, like the keyboard.